The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 13:10:17.799 UTC [46] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:10:17.808 UTC [47] LOG:  database system was shut down at 2025-02-14 13:10:17 UTC
2025-02-14 13:10:17.811 UTC [46] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 13:10:18.162 UTC [46] LOG:  received fast shutdown request
2025-02-14 13:10:18.163 UTC [46] LOG:  aborting any active transactions
2025-02-14 13:10:18.164 UTC [46] LOG:  background worker "logical replication launcher" (PID 53) exited with exit code 1
2025-02-14 13:10:18.164 UTC [48] LOG:  shutting down
2025-02-14 13:10:18.171 UTC [46] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:10:18.270 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:10:18.270 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:10:18.271 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:10:18.277 UTC [64] LOG:  database system was shut down at 2025-02-14 13:10:18 UTC
2025-02-14 13:10:18.281 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:10:24.507 UTC [79] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:10:24.507 UTC [79] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:10:24.879 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:10:24.879 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_da19e29f-1d5d-400b-ad37-808299a5766a" WITH (FORCE)
2025-02-14 13:10:25.379 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:10:25.379 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_73fbcdd5-85b6-4d7d-a245-af0e0f4576a2" WITH (FORCE)
2025-02-14 13:10:25.783 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:10:25.783 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_269f8d65-6051-4e25-b806-90864833bc42" WITH (FORCE)
2025-02-14 13:10:26.219 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:10:26.219 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_858d07e1-b59d-44b0-be02-4bbe9f808a05" WITH (FORCE)
